sentry-android-core/src/main/java/io/sentry/android/core/performance/AppStartMetrics.java

```diff
@@ -346,6 +347,13 @@ public void onActivityCreated(@NonNull Activity activity, @Nullable Bundle saved
 
     // the first activity determines the app start type
     if (activeActivitiesCounter.incrementAndGet() == 1 && !firstDrawDone.get()) {
+
```

Comment:
oh, I just realized - we only set the currentActivity for the first activity here right? So it won't be the "last known activity" but rather the first one, which is incorrect, right? E.g. if you launched 3 activities, and then initialized the SDK it would use the wrong one to hook user interactions. Unless I got the above if-check wrong 😅